src/components/Search/Search.tsx

```diff
@@ -32,22 +42,42 @@ const Search = ({ altText = 'Search', useIcon = true }: React.PropsWithChildren<
       </button>
       <form
         data-testid="search-form"
-        className={classnames(
-          `${px}-search__input-wrapper`,
-          { [`${px}-search__input-wrapper--active`]: overlayEnabled },
-          { [`${px}-search__input-wrapper--use-icon`]: useIcon },
-        )}
+        className={classnames(`${px}-search__form`, { [`${px}-search__form--active`]: overlayEnabled })}
       >
-        {useIcon ? (
-          <SearchIcon data-testid={`search-form-icon`} className={`${px}-search__input-wrapper__icon`} />
-        ) : null}
-        <Input
-          className={`${px}-search__input`}
-          alt={altText}
-          hideLabel={true}
-          placeholder={`Type to search`}
-          type="text"
-        />
+        <div
+          className={classnames(`${px}-search__input-wrapper`, { [`${px}-search__input-wrapper--use-icon`]: useIcon })}
+          role="combobox"
+          aria-haspopup="listbox"
+        >
+          {useIcon ? (
+            <SearchIcon data-testid={`search-form-icon`} className={`${px}-search__input-wrapper__icon`} />
+          ) : null}
+          <Input
+            className={`${px}-search__input`}
+            alt={altText}
+            hideLabel={true}
+            placeholder={`Type to search`}
+            type="text"
+            onChange={
+              onSearch
+                ? (e: { target: { value: string } }) => {
+                    console.log('e.target.value', e.target.value);
+                    onSearch(e.target.value);
+                  }
+                : undefined
+            }
+            ref={searchInputRef}
+          />
+          {searchInputRef.current?.value && searchInputRef.current?.value.length > 2 ? (
+            <SearchResults autoCompleteResults={searchResults}>
+              <li key="viewAllSearchResults" className={`${px}-search__result`}>
+                <Link href={`/Search?Search=${encodeURIComponent(searchInputRef.current?.value)}`}>
+                  <p>{`View all results for ${searchInputRef.current?.value}`}</p>
```

Comment:
I think this `view all results for` needs to be translatable.  Unfortunately it'll have to be a callback function, not a direct string because the "value" will need to be put in different locations in the string translations